src/components/SecondaryPanes/SecondaryPanes.css

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 .secondary-panes {
-  overflow: auto;
+  overflow-x: hidden;
```

Comment:
Turns out that using `overflow-x: hidden` automatically sets `overflow-y: auto`.
But if we were to remove that `overflow-x` declaration, then `overflow-y` would revert to `visible`, which we don't want.
Can you add an explicit `overflow-y: auto;` here to prevent that and better communicate our intent?

Reply:
I didn't think about that! Added :)